src/grafana_agent.py

```diff
@@ -199,6 +206,12 @@ def _on_cloud_config_revoked(self, _) -> None:
         logger.info("cloud config revoked")
         self._update_config()
 
+    def _on_cert_transfer_available(self, event):
+        # TODO: trust the certificate instead of printing it
```

Comment:
If we do "delta-charming" here, then I imagine that with a unique filename such as `model-model_uuid-relation_name-relation_id` we should be able to also delete the file on relation broken.